.gitignore

```diff
@@ -55,6 +55,8 @@ config/settings.local.yml
 config/settings/*.local.yml
 config/environments/*.local.yml
 
+config/supported_ssg.yaml
```

Comment:
This doesn't seem to work as expected. Perhaps because it's already checked in to git?